tasks/new_e2e_tests.py

```diff
@@ -288,13 +288,21 @@ def _destroy_stack(ctx: Context, stack: str):
             env=_get_default_env(),
         )
         if ret is not None and ret.exited != 0:
+            if "No valid credential sources found" in ret.stdout:
+                print(
+                    "No valid credentials sources found, you need to wrap the invoke command in aws-vault exec <account> --"
+                )
+                exit(1)
```

Comment:
```suggestion
                raise Exit(color_message("No valid credentials sources found, you need to wrap the invoke command in aws-vault exec <account> --", Color.RED), code=1)
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏For a cleaner output, feel free to ignore.

Reply:
applied